<?xml version="1.0"?>
<launch>

    <node pkg="whud_serial_bridge" type="serial_bridge_node" name="serial_bridge_node" output="screen">
        <param name="serial_port_path" type="string" value="/dev/ttyTCU0"/>
        <param name="serial_baundrate" type="int" value="115200"/>
    </node>

</launch>